Platform/TigerlakeBoardPkg/BoardConfig.py

```diff
@@ -165,8 +165,20 @@ def __init__(self, *args, **kwargs):
         self.FWUPDATE_SIZE        = 0x00020000 if self.ENABLE_FWU else 0
         self.OS_LOADER_FD_NUMBLK  = self.OS_LOADER_FD_SIZE // self.FLASH_BLOCK_SIZE
 
-        self.TOP_SWAP_SIZE        = 0x080000
-        self.REDUNDANT_SIZE       = 0x360000
+        self.UCODE_SLOT_SIZE         = 0x0001C000
+
+        self.BUILD_IDENTICAL_TS      = 1
+        self.ENABLE_SBL_RESILIENCY   = 1
```

Comment:
Will want SBL resiliency disabled whenever FSP debug is enabled. Might also want SG1B size to depend on if FSP debug is enabled.